routes/auth.js

```diff
@@ -0,0 +1,11 @@
+const express=require("express");
+
+const router=express.Router();
+
+const { signup, signin, signout }=require("../controllers/auth")
+
+router.post("/signup",signup);
+router.post("signin",signin);
+router.get("signout",signout);
+
+module.exports=router;
```

Comment:
Don't make a new auth file for these routes and add them in index.js itself

Reply:
added the routes in index.js